CMakeLists.txt

```diff
@@ -78,34 +79,38 @@ endif()
 
 message(STATUS "CMAKE_SYSTEM_PROCESSOR: ${CMAKE_SYSTEM_PROCESSOR}")
 
-if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm" OR ${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm" OR ${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
     message(STATUS "ARM detected")
 else()
     message(STATUS "x86 detected")
-    if (MSVC)
+
+    if(MSVC)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /arch:AVX2")
         set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /arch:AVX2")
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /arch:AVX2")
     else()
         if(NOT LLAMA_NO_AVX)
             set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mavx")
         endif()
+
         if(NOT LLAMA_NO_AVX2)
             set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mavx2")
         endif()
+
         if(NOT LLAMA_NO_FMA)
             set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mfma")
         endif()
+
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mf16c")
     endif()
 endif()
 
 # if (LLAMA_PERF)
-#     set(LLAMA_EXTRA_FLAGS ${LLAMA_EXTRA_FLAGS} -DGGML_PERF)
+# set(LLAMA_EXTRA_FLAGS ${LLAMA_EXTRA_FLAGS} -DGGML_PERF)
 # endif()
-
 add_executable(llama
     main.cpp
+    llama.cpp
```

Comment:
llamalib already contains llama.cpp, utils.cpp and utils.h

Reply:
Updated.